components/tensorflow-notebook-image/Dockerfile

```diff
@@ -50,8 +50,11 @@ ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US.UTF-8
 
 # Create jovyan user with UID=1000 and in the 'users' group
-RUN useradd -m -s /bin/bash -N -u $NB_UID $NB_USER && \
-    chown -R ${NB_USER}:users /usr/local/bin
+# but allow for non-initial launches of the notebook to have
+# $HOME provided by the contents of a PV
```

Comment:
What are we doing to allow non-initial launches of the notebook to have HOME provided by contents of a PV?

Reply:
See below.